lib/api/controllers/documentController.js

```diff
@@ -146,9 +146,13 @@ class DocumentController extends NativeController {
    * @returns {Promise<Object>}
    */
   mGet (request) {
-    const
-      ids = this.getBodyArray(request, 'ids'),
-      { index, collection } = this.getIndexAndCollection(request);
+    let ids;
+    if (request.input.body && Object.keys(request.input.body).length) {
```

Comment:
This request will make mGet fail:

```
{
  "controller": "document",
  "action": "mGet",
  "body": { "foo": "bar" },
  "ids": [ "baz", "qux" ],
  "index": "foo",
  "collection": "bar"
}
```

(use case example: a plugin applying a post-get filter, provided in the body part of the request)